ui/admin/tests/acceptance/onboarding/onboarding-test.js

```diff
@@ -0,0 +1,43 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { authenticateSession } from 'ember-simple-auth/test-support';
+import { visit, find, fillIn, click, currentURL } from '@ember/test-helpers';
+
+module('Acceptance | onboarding', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  const urls = {
+    onboarding: '/onboarding',
+    success: '/onboarding/success',
+    orgs: '/scopes/global/scopes',
+  };
+
+  hooks.beforeEach(() => {
+    authenticateSession({});
+  });
+
+  test('show targetAddress and targetPort fields', async function (assert) {
+    assert.expect(2);
+    await visit(urls.onboarding);
+    assert.ok('[name="targetAddress"]');
+    assert.ok(find('[name="targetPort"]'));
```

Comment:
Can we update these tests to use the `assert.dom('selector').isVisible()` pattern?

Reply:
@cameronperera sorry I forgot about this comment. Changed! thanks for the catch